src/Logs/Monolog/src/Handler.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Logs\Monolog;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Formatter\NormalizerFormatter;
+use Monolog\Handler\AbstractProcessingHandler;
+use OpenTelemetry\API\Logs as API;
+
+class Handler extends AbstractProcessingHandler
+{
+    private API\LoggerInterface $logger;
+
+    /**
+     * @psalm-suppress InvalidArgument
+     */
+    public function __construct(API\LoggerProviderInterface $loggerProvider, $level, bool $bubble = true)
+    {
+        parent::__construct($level, $bubble);
+        $this->logger = $loggerProvider->getLogger('monolog');
+    }
+
+    protected function getDefaultFormatter(): FormatterInterface
+    {
+        return new NormalizerFormatter('Uu');
```

Comment:
```suggestion
        return new NormalizerFormatter();
```
I don't think we should change the dateFormat as it affects datetimes in `context` and `extra` too; I think we should populate timestamp, severity and channel from the original record and use the formatted record for body, context and extra.